src/sentry/tasks/post_process.py

```diff
@@ -411,6 +411,17 @@ def fetch_buffered_group_stats(group):
     group.times_seen_pending = result["times_seen"]
 
 
+def fetch_buffered_group_last_seen(event, group):
+    """
+    Fetches buffered updates to `last_seen` for this group.
+    """
+    from sentry import buffer
+    from sentry.models import Group
+
+    result = buffer.get(Group, ["last_seen"], {"pk": group.id})
+    group.last_seen_buffered = max(result["last_seen"], int(datetime.timestamp(event.datetime)))
```

Comment:
I'd probably recommend using `group.last_seen` as the fallback so you don't have to use an event